IndividualSpreadFooting.tex

```diff
@@ -372,27 +372,27 @@ \section{Design for Moment and Beam Shear}
          \label{eq:concretesolidslabs}
         \sage{tau_a}=\sage{z}
 \end{equation}
-$\sage{tau_c}$ is given by \tablemacro 19 of the  \citetitle{IS2000} 
+$\sage{tau_c}$ is given by \tablem 19 of the  \citetitle{IS2000} 
 depending on the steel area provided for moment at the critical section
 2-2 (the minimum value of $\sage{tau_c}$ is assumed for 
-$p_t $$\leq$$ 0.15$ in \tablemacro 19 and
+$p_t $$\leq$$ 0.15$ in \tablem 19 and
 
 $$\begin{array}{l@{}l}
-&{}k=1.0 \text{ for } D \geq 300 \cmmacro\\
-&{}=1.1 \text{ for } D = 250 \cmmacro\\   
-&{}=1.2 \text{ for } D = 200 \cmmacro\\
-&{}=1.25 \text{ for } D = 175 \cmmacro\\   
-&{}=1.30 \text{ for } D \leq 150 \cmmacro
+&{}k=1.0 \text{ for } D \geq 300 \cm\\
+&{}=1.1 \text{ for } D = 250 \cm\\   
+&{}=1.2 \text{ for } D = 200 \cm\\
+&{}=1.25 \text{ for } D = 175 \cm\\   
+&{}=1.30 \text{ for } D \leq 150 \cm
 \end{array}$$
 Normally, depth of footing given for perimeter shear
-(\chartmacro \ref{Dummy chart}) is more than adequate
+(\chartm \ref{Dummy chart}) is more than adequate
 to satisfy the requirements of beam shear. But when steel type 
-\Fefouronefivemacro and \Fefivezerozeromacro are used as reinforcement
+{\fefouronefive} and {\fefivezerozero} are used as reinforcement
```

Comment:
It's Okay.

See if these help:
http://tex.stackexchange.com/questions/9718/defining-commands-abbreviations-that-contain-numbers
http://tex.stackexchange.com/questions/28915/number-in-name-of-commands